src/stavenote.ts

```diff
@@ -121,6 +121,10 @@ export class StaveNote extends StemmableNote {
   static get DEFAULT_LEDGER_LINE_OFFSET(): number {
     return 3;
   }
+  static get minNoteheadPadding() {
```

Comment:
The return value is missing, make it:
`static get minNoteheadPadding(): number`

Reply:
Right!